crates/sui-graphql-rpc/src/types/query.rs

```diff
@@ -615,6 +616,27 @@ impl Query {
             .await
             .extend()
     }
+
+    /// Fetch a list of objects by their IDs and versions.
+    async fn multi_get_objects(
```

Comment:
Would it make sense to allow fetching objects just by ids (so the latest version?)